featurebyte/query_graph/graph.py

```diff
@@ -138,7 +138,31 @@ def get_entity_ids(self, node_name: str) -> List[ObjectId]:
                     output.extend(node.parameters.entity_ids)
             elif isinstance(node, LookupNode):
                 output.append(node.parameters.entity_id)
-        return sorted(output)
+        return sorted(set(output))
```

Comment:
We should remove duplicated items from the output list. Add tests to cover above issue.